### PR TITLE
Add id to a11y elements

### LINKED ISSUE #272: AccessibilityView should be Identifiable
Identifiable support would allow use of ForEach in SwiftUI.

Also it's just good to be able to identify element or container somehow.

Label is not an id, frame too.

### PR DESCRIPTION
**Pull request by AgapovOne**
Closes #272

### CODE REVIEW

**Review by akaDuality — APPROVED** [review]

**Comment by akaDuality** [comment]
@AgapovOne Can you run tests locally and fix compilation errors?

**Comment by AgapovOne** [comment]
Those are compilation errors on Xcode 14. Xcode 15 runs all these fine. 

Can you update xcode on CI? 

@akaDuality

File: Shared/TestHelpers/DocumentTestHelpers/A11yDescription+TestMake.swift [hunk 1]
```diff
@@ -9,6 +9,7 @@ import Foundation
 
 extension A11yDescription {
     public static func testMake(
+        id: UUID = .init(),
```
Comment (akaDuality) [inline Shared/TestHelpers/DocumentTestHelpers/A11yDescription+TestMake.swift:14]: Cool, default value allows to stub reproducible id for snapshots in the future

**Review by akaDuality — APPROVED** [review]

**Comment by akaDuality** [comment]
Updated Xcode to 15 on CI, thx

**Pull request merged**